### ISSUE TITLE
Log displays incorrect build as culprit

### ISSUE THREAD

**Issue by Ebola16** [Helios747/DolphinBisectTool#23]
I was investigating https://bugs.dolphin-emu.org/issues/10563 and verified https://dolphin-emu.org/download/dev/master/5.0-76/ is the culprit PR. DBT correctly identified 5.0-76 as the culprit in its notice message. The log however displays an incorrect PR.

![capturex](https://user-images.githubusercontent.com/17330088/31107311-97df630c-a7bf-11e7-9bda-f6a2f7a4a877.PNG)

**Comment by Helios747** [comment]
augh, off by one errors. Thanks. I'll look into this later.

**Comment by niconii** [comment]
Weirdly enough, I just had the opposite experience, where the version reported in the log was the culprit, not the one displayed by the program. Either way, something's definitely not right.

**Comment by Helios747** [comment]
Yeah. Eventually I want to get around to rewriting this in a more portable language and hope to break down some of this syntactic sugar that is masking these bugs.

**Comment by Ebola16** [comment]
I just had the opposite problem with version 2.0.0 now. Yay.
